LinaSaysHello/Program.cs

```diff
@@ -13,16 +13,54 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            //Console.WriteLine("Hi! What’s your name?");
-            //string name = Console.ReadLine();
-            //Console.WriteLine("Nice to meet you " + name);
-            //Console.WriteLine("Anastasiya");
-            //Console.WriteLine("Vention");
-            //Console.ReadLine();
+            //Task1//
+            int num = 20;
+            num += 5;
 
-           
+            Console.WriteLine("Variable:" + num);
 
+            //Task2//
 
+            Console.Write("Enter a 4 digit number of days: ");
+            int number = Convert.ToInt32(Console.ReadLine());
+            int years = number / 365;
+            int months = (number - years * 365) / 30;
+            int days = (number - years * 365) - months * 30;
+
+            Console.WriteLine("This number represents: " + years + " years " + months + " months " + days + " days");
+
+            //Task3//
+            
+            Console.Write("Enter a random number: ");
+            int randomNumber = Convert.ToInt32(Console.ReadLine());
+            int result = (randomNumber + randomNumber * 2);
```

Comment:
It's worth using var keyword instead of declaring the explicit type of the variable. Because for example in this case, if you set an explicit type, you will have to create several variables to output the result.